amida-tech/greyscale
# Task/inba 370 user integration test

#### What's this PR do?
Fixes the `user.integration` test to now work with a mock of the auth service. 

#### Related JIRA tickets:
[370](https://jira.amida-tech.com/browse/INBA-370)

#### How should this be manually tested?
- Run the `user.intgration` test and make sure all test pass:
   from `greyscale/backend` run `node_modules/.bin/mocha test/user.integration.js --bail -t 10000`
- To do a little more thorough testing you can:
   - Create the same user that is created when you run `user.integration` on the auth service
      ``` email: test-adm@mail.net
       username: test-adm@mail.net
       scope: admin
       password: testadmin12345
   - Log in on the auth service with that user and use the JWT to access protected routes on 
      greyscale. 
   - Do this for a user that doesn't exist on greyscale (it should fail) etc. This helps verify that the 
    JWTStrategy middleware on greyscale works as expected

#### Any background context you want to provide?

#### Screenshots (if appropriate):
